pkgs/tools/networking/openntpd/default.nix

```diff
@@ -17,6 +17,7 @@ stdenv.mkDerivation rec {
     "--with-privsep-user=${privsepUser}"
     "--sysconfdir=/etc"
     "--localstatedir=/var"
+    "--with-cacert=/etc/ssl/certs/ca-certificates.crt"
```

Comment:
Could you test and see if this still works with `"${cacerts}/etc/ssl/certs/ca-bundle.crt"`. That would solve the problem on other operating systems.

Reply:
I don't believe it would, because the `cacerts` package contains a fixed set of standard certificates, but other operating systems may have a different set of certificates, and the user may even have added (or removed) their own certificates to the ones that came with their operating system.